schema/src/main/jade-tables/experiment_activity.table.json

```diff
@@ -17,14 +17,8 @@
       "type": "required"
     },
     {
-      "name": "time_started",
-      "datatype": "timestamp",
-      "type": "required"
-    },
-    {
-      "name": "time_ended",
```

Comment:
As far as I could see in the data and the mapping sheet, ENCODE doesn't publish this information for experiments, so there's not much we can do.